dts/bindings/can/xlnx,can.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2022, Weidmueller Interface GmbH & Co. KG
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: |
+  Xilinx Processor System CAN controller
+
+  This CAN controller is contained in both the Xilinx Zynq-7000 and
+  ZynqMP (UltraScale) SoCs.
+
+compatible: "xlnx,can"
+
+include: can-controller.yaml
+
+properties:
+    reg:
+      required: true
+
+    interrupts:
+      required: true
+
+    clock-frequency:
+      type: int
+      required: true
+      description: |
+        Specifies the base clock frequency from which the CAN controller's
+        clock frequency for the selected baud rate will be derived using
+        a divider in the respective CAN controller's Baud Rate Prescaler
+        Register. Therefore, the value of this item must be set to the clock
+        frequency of the PLL supplying the respective CAN controller's clock
+        - by default, this is the IO PLL.
```

Comment:
Can't this frequency be established using the clock controller framework and devicetree?